src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -179,6 +185,49 @@ function useSidebarMetadataFetcher(
         [api, file, onApiError],
     );
 
+    const extractSuggestions = React.useCallback(
+        async (templateKey: string, fields: MetadataTemplateField[]) => {
+            const aiAPI = api.getIntelligenceAPI();
+
+            const oldToken = api.xhr.token;
+            api.xhr.token = async function overwrittenToken(...args) {
+                const tokenOjb = await oldToken(...args);
+
+                return {
+                    read: tokenOjb.read,
+                };
+            };
+
+            let answer = {};
+            try {
+                answer = await aiAPI.extractStructured({
+                    items: [file],
+                    metadata_template: {
+                        scope: 'enterprise',
```

Comment:
should this scope reflect `template.scope` ?

Reply:
Good point. Although `enterprise` as a value is working, and the ID is inferred by backend, some of the templates can have `global` scope.
I'll move to use `fields`. As that will not require any changes to `metadata-editor` and still works.